modules/end-user-guide/partials/proc_creating-a-workspace-from-template-devfile.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+
+[id="creating-a-workspace-from-template-devfile_{context}"]
+= Creating a workspace from template devfile
+
+You can create a new workspace by using one of the devfile templates included in {prod-short}.
+
+.Prerequisites
+
+* A running instance of {prod-short}
++
+To install {prod-short}, see the xref:installation-guide:installing-che.adoc[Installation Guide].
+
+.Procedure
+
+. Go to the xref:navigating-che-using-the-dashboard.adoc[Dashboard] of {prod-short}.
+
+. In the left navigation panel, click btn:[Create Workspace] to go to the *Custom Workspace* tab.
+
+. Enter a *Workspace Name* for the workspace that you are creating.
```

Comment:
can't you just say "enter a workspace name" without the markup?

Reply:
Yep, I should have rewritten it earlier.